MapleServer2/Types/DamageHandler.cs

```diff
@@ -39,20 +39,20 @@ public DamageSourceParameters()
 
 public class DamageHandler
 {
```

Comment:
Not sure how I feel about a Damager Handler being in Types.

Reply:
Where would you recommend moving it to?